scripts/recan_bundle_tdr.py

```diff
@@ -281,20 +263,26 @@ def dump_tables(bundle: TDRBundle) -> MutableJSON:
     }
 
 
-def add_supp_files(bundle: TDRBundle, source_uuid: str, *, num_files: int) -> None:
+def add_supp_files(bundle: TDRBundle,
+                   source_uuid: str,
+                   *,
+                   num_files: int,
+                   version: str
+                   ) -> None:
     links_json = bundle.metadata_files['links.json']['links']
     links_manifest = one(e for e in bundle.manifest if e['name'] == 'links.json')
     project_id = bundle.metadata_files['project_0.json']['provenance']['document_id']
 
-    metadata_ids = sorted(random_deterministic_uuid(bundle.uuid) for _ in range(num_files))
+    # UUIDs and document names must BOTH be sorted for consistency with plugin.
```

Comment:
They can't both be sorted. Sorting by one yields a different ordering than sorting by the other. Whatever it is you're trying to say is not clear to me.

Reply:
What I'm trying to say is: when the plugin retrieves entities from bigquery, it sorts them by UUID, and then enumerates them to generate the numerical component of the document names (e.g. the "2" in "cell_supension_2.json"). So both the entity UUIDs and the document names _are_ sorted, albeit not by the same key (document names are ordered numerically, not lexicographically). So, the script needs to follow the same logic, obtain a sequence of sorted UUIDs and then enumerate then, to generate the expected document names. Comment has been updated with an attempt to express this more succinctly.